### ISSUE TITLE
No way to set fcolor back to base color setting

### ISSUE THREAD

**Issue by dconner4** [darmuh/TerminalStuff#9]
After setting the flashlight color in the terminal to any other applicable color, it doesn't seem to be possible to revert that change back to the base color that it was previously.

Looking inside the switch statement at https://github.com/darmuh/TerminalStuff/blob/2.0.X/DarmuhsTerminalCommands/MyTerminalPatcher.cs#L60
there is no case for white or default of whatever the normal color is so it gets set to null which will return a failure prompt in the terminal

https://github.com/darmuh/TerminalStuff/blob/2.0.X/DarmuhsTerminalCommands/MyTerminalPatcher.cs#L437

Would be great to run a command at the terminal to see a list of the available colors as the "sasstro" color was quite nice but only found out about it after reading the code.

**Comment by darmuh** [comment]
good suggestions! I think i'll try to push both of these in the next patch as long as I dont run into any issues.

As for now, you can see which colors are implemented in a portion of the readme, though I know most people dont read that lol

**Comment by darmuh** [comment]
I've created a list command to see available colors via "fcolor list" that will be in the next patch.

I've also added a default/normal setting for when you want to set it back to white/normal.

I also changed how the flashlight color changes because the old version was having issues setting the color until the player had already turned the light on and switched to helmet light. It now should work at any point as long as you have a flashlight in your inventory. I've also confirmed this works with FlipMods' "ReservedFlashlightSlot" Mod.
